src/components/Footer/Footer.js

```diff
@@ -1,94 +1,111 @@
-import React from 'react'
-import styled from 'styled-components'
-import { GiMedicines} from 'react-icons/gi'
-import { GrFacebookOption } from 'react-icons/gr'
-import { AiOutlineInstagram } from 'react-icons/ai'
-import { RiLinkedinFill } from 'react-icons/ri'
-import { CgTwitter } from 'react-icons/cg'
-
-
+import React from "react";
+import styled from "styled-components";
+import { GiMedicines } from "react-icons/gi";
+import { GrFacebookOption } from "react-icons/gr";
+import { AiOutlineInstagram } from "react-icons/ai";
+import { RiLinkedinFill } from "react-icons/ri";
+import { CgTwitter } from "react-icons/cg";
 
 const MainFooter = styled.div`
-position: absolute;
-width: 100%;
-bottom: 0;
-`
-const Container = styled.div`
-`
+  position: absolute;
+  width: 100%;
+  height: 200px;
+  bottom: 0;
+`;
+const Container = styled.div``;
 const Col = styled.div`
-display: flex;
-flex-direction: column;
-`
+  display: flex;
+  flex-direction: column;
+`;
 
 const Row = styled.div`
-display: flex;
-justify-content:  space-around;
-`
+  display: flex;
+  justify-content: space-around;
+`;
 const SocialIcons = styled.div`
-display: flex;
-justify-content: space-around;
-padding: 30px 0;
-font-size: 1.5rem;
-color: #15A9E3;
-`
-
-
+  display: flex;
+  justify-content: space-around;
+  padding: 30px 0;
+  font-size: 1.5rem;
+  color: #15a9e3;
+`;
 
 const Left = styled.div`
-flex: 1;
-display: flex;
-align-items: center;
-
-`
+  flex: 1;
+  display: flex;
+  align-items: center;
+`;
 
 export default function Footer() {
   return (
     <MainFooter>
-     <Container>
+      <Container>
+        <Row>
+          <Col>
+            <Left>
+              <GiMedicines
+                size={25}
+                style={{ paddingRight: 8, color: "#15A9E3" }}
+              />
+              <h2 style={{ color: "#184363" }}>Happy Medi</h2>
```

Comment:
As I mentioned in the previous PR, this type of styling is barely maintainable when the application grows. In addition, the ability of responsiveness drops whe you do static styling. Please consider to use classNames or Ids and do the stylings in the stylesheets accordingly.